cpg_utils/config.py

```diff
@@ -1,14 +1,14 @@
 """Provides access to config variables."""
 
 import os
-from typing import Any, MutableMapping, Optional
+from typing import Optional
 from cloudpathlib import AnyPath
 import toml
 
 # We use these globals for lazy initialization, but pylint doesn't like that.
 # pylint: disable=global-statement, invalid-name
 _config_path = os.getenv('CPG_CONFIG_PATH')  # See set_config_path.
-_config: Optional[MutableMapping[str, Any]] = None  # Cached config, initialized lazily.
+_config: Optional[dict] = None  # Cached config, initialized lazily.
```

Comment:
I prefer `Dict` from typing, then you can annotate it like `Dict[str, Any]`, but this is fine too

Reply:
@illusional you can use `dict[str, Any]`

Reply:
Ah that's a Python 3.10 thing I see